minimisers/generalUtils/makeEmptyBayesResultsStruct.m

```diff
@@ -91,16 +91,16 @@
     
     if isDomains
         sldXdata = cell(nContrasts,2);
-        sldDataCell2 = [1 1 1 ; 1 1 1];
-        coder.varsize('sldDataCell',[1 1e4],[1 1]); 
+        sldDataCell = [1 1 1; 1 1 1];
+        coder.varsize('sldDataCell',[2 1e4],[1 1]); 
```

Comment:
It was solving this bug (the use of "sldDataCell" and "sldDataCell2") that caused/exposed the other problems here.